setup.py

```diff
@@ -24,7 +24,7 @@ def read(*parts):
     url='https://github.com/python-babel/django-babel/',
     packages=find_packages(exclude=('tests',)),
     install_requires=[
-        'django>=1.4,<1.10',
+        'django>=1.4,<1.11',
         'babel>=1.3',
     ],
     classifiers=[
```

Comment:
Sure, but then I'd have to add another package anyway to contain the bits that need to be in a Python package/Django app.

I've used this sort of setup elsewhere for Django projects and it seems to work okay.

Do you want me to change this anyway?